dashboard/config.ini

```diff
@@ -10,6 +10,7 @@ pydoc_job=devtools-fabric8-analytics-common-fabric8-analytics-pydoc
 build_job=devtools-fabric8-analytics-data-model-f8a-build-master
 test_job=devtools-fabric8-analytics-data-model-fabric8-analytics
 pylint_job=devtools-fabric8-analytics-data-model-fabric8-analytics-pylint
+pydoc_job=https://ci.centos.org/job/devtools-fabric8-analytics-jobs-fabric8-analytics-pydoc
```

Comment:
I'd say `pydoc_job=devtools-fabric8-analytics-data-model-fabric8-analytics-pydoc`

Reply:
you are right, bad /me